composer.json

```diff
@@ -21,5 +21,20 @@
         "os2forms/os2forms": "^2.6",
         "os2forms/os2forms_forloeb": "^1.7",
         "os2web/os2web_simplesaml": "8.x-dev"
+    },
+    "extra": {
+        "composer-exit-on-patch-failure": true,
+        "patchLevel": {
+            "test": "-p2"
+        },
+        "enable-patching": true,
+        "patcher": {
+            "force-reset": true
+        },
+        "patches": {
+            "drupal/dynamic_entity_reference": {
+                "entityQuery reference JOINs should specify target_type": "https://www.drupal.org/files/issues/2021-06-22/entityquery-reference-joins-should-specify-target_type-3120952-24.patch"
```

Comment:
Consider adding the url to the actual issue in the patch title for easy reference, i.e.

```
  "entityQuery reference JOINs should specify target_type (https://www.drupal.org/project/dynamic_entity_reference/issues/3120952#comment-14141038)": "…"
```